modules/debugging-issues-with-rhel-tools.adoc

```diff
@@ -11,10 +11,9 @@ debugging scaling or performance problems. This container image:
 * Allows users to deploy minimal footprint container hosts by moving packages out
 of the base distribution and into this support container.
 
-* Provides debugging capabilities for Red Hat Enterprise Linux 7 Atomic Host,
-which has an immutable package tree. `rhel-tools` includes utilities such as
-tcpdump, sosreport, git, gdb, perf, and many more common system administration
-utilities.
+* Provides debugging capabilities for Red Hat Enterprise Linux CoreOS, which has
+an immutable package tree. `rhel-tools` includes utilities such as tcpdump,
+sosreport, git, gdb, perf, and many more common system administration utilities.
```

Comment:
Unfortunately, despite what the description of the rhel-tools image says, sosreport (provided by the sos package) is no longer included in the image.

Reply:
@control-d Thank you for catching this! I opened https://github.com/openshift/openshift-docs/pull/14577 to remove the reference.